frame/contracts/src/wasm/prepare.rs

```diff
@@ -343,12 +343,6 @@ impl<'a, T: Config> ContractModule<'a, T> {
 				.get(*type_idx as usize)
 				.ok_or_else(|| "validation: import entry points to a non-existent type")?;
 
-			// We disallow importing `seal_println` unless debug features are enabled,
-			// which should only be allowed on a dev chain
-			if !self.schedule.enable_println && import.field().as_bytes() == b"seal_println" {
-				return Err("module imports `seal_println` but debug features disabled");
-			}
-
```

Comment:
Nice so now we can keep the debug statements when deploying to a non dev chain :heart: 

Good for the contract dev cycle and deploying to testnets etc.

Reply:
Yes. Looking forward to reviewing the ink implementation :)